sdk/nodejs/types/output.ts

```diff
@@ -6488,7 +6542,7 @@ export namespace ec2 {
 
     export interface GetLaunchTemplateNetworkInterface {
         associatePublicIpAddress?: boolean;
-        deleteOnTermination: boolean;
+        deleteOnTermination: string;
```

Comment:
Is this change expected?

Reply:
This is not - we need to override this as its due to their "nullable bool" approach